app/helpers/blacklight/catalog_helper_behavior.rb

```diff
@@ -172,12 +172,13 @@ def has_thumbnail? document
   ##
   # Render the thumbnail, if available, for a document and
   # link it to the document record.
-  # 
+  #
   # @param [SolrDocument]
   # @param [Hash] options to pass to the image tag
   # @param [Hash] url options to pass to #link_to_document
   # @return [String]
   def render_thumbnail_tag document, image_options = {}, url_options = {}
+    return "" unless has_thumbnail?(document) # do not render any html if there is no thumbnail
```

Comment:
In the view we check the `has_thumbnail?` helper and the truthy-ness of the return value of this method (`""` is truthy).  Now we're also checking the `has_thumbnail?` helper in this helper as well.

I feel like we should reconcile this somehow so we're not redundantly checking the `has_thumbnail?` helper and ensure we are not checking the truthy-ness of an empty string.

Just my $.02

https://github.com/projectblacklight/blacklight/blob/master/app/views/catalog/_thumbnail_default.html.erb#L1